plugins/ui/src/deephaven/ui/components/toast.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import annotations
+
+from ..hooks import use_event_queue
+
+from typing import Callable
+from .make_component import make_component
+from .._internal.utils import dict_to_react_props
+from ..types import ToastVariant
+
+_TOAST_EVENT = "toast.event"
+
+
+@make_component
+def toast(
+    message: str,
+    variant: ToastVariant = "neutral",
```

Comment:
We should treat the rest of the arguments as keyword args only:
```suggestion
    *,
    variant: ToastVariant = "neutral",
```